tests/lib/util/npm-utils.js

```diff
@@ -162,8 +162,7 @@ describe("npmUtils", () => {
         });
 
         it("should return false if package.json does not exist", () => {
-            mockFs({});
-            assert.strictEqual(npmUtils.checkPackageJson(), false);
+            assert.strictEqual(npmUtils.checkPackageJson("/"), false);
```

Comment:
Will this also work on windows where `/` is not the root directory?

Reply:
I don't use Windows, so it's not failing for me.
I was just curious whether this is working as intended. Looks like it does, thanks.